ethereum/spec/src/main/java/tech/pegasys/teku/spec/Spec.java

```diff
@@ -461,9 +461,11 @@ public boolean verifyProposerSlashingSignature(
   public boolean verifyVoluntaryExitSignature(
       BeaconState state, SignedVoluntaryExit signedExit, BLSSignatureVerifier signatureVerifier) {
     final UInt64 epoch = signedExit.getMessage().getEpoch();
+    final Bytes32 voluntaryExitDomain =
+        getBlockProcessor(state.getSlot()).computeVoluntaryExitDomain(signedExit, state);
```

Comment:
Hmm actually I had a second look and think this should use the SpecVersion from the signedExit epoch so can do 

```java
final SpecVersion specVersion = atEpoch(epoch);
final Bytes32 domain =
        specVersion.getBlockProcessor().computeVoluntaryExitDomain(signedExit, state);
return specVersion.operationSignatureVerifier()....
```

Reply:
Done